Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/Adaptive_sizing_field.h

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) 2023 GeometryFactory (France)
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Ivan Paden
+
+#ifndef CGAL_PMP_REMESHING_ADAPTIVE_SIZING_FIELD_H
+#define CGAL_PMP_REMESHING_ADAPTIVE_SIZING_FIELD_H
+
+#include <CGAL/license/Polygon_mesh_processing/meshing_hole_filling.h>
+
+#include <CGAL/Polygon_mesh_processing/internal/Sizing_field_base.h>
+#include <CGAL/Polygon_mesh_processing/interpolated_corrected_curvatures.h>
+
+#include <CGAL/boost/graph/selection.h>
+#include <CGAL/boost/graph/Face_filtered_graph.h>
+
+#include <CGAL/number_utils.h>
+
+namespace CGAL
+{
+namespace Polygon_mesh_processing
+{
+/*!
+* \ingroup PMP_meshing_grp
+* provides criteria for isotropic remeshing to achieve variable mesh edge lengths as a function
+* of local discrete curvatures.
+*
+* The local discrete curvatures are calculated using the
+* `CGAL::Polygon_mesh_processing::interpolated_corrected_curvatures()` function.
+*
+* Edges longer than the local target edge length are split in two, while
+* edges shorter than the local target edge length are collapsed.
```

Comment:
```suggestion
* Edges too long with respect to the local target edge length are split in two, while
* edges that are too short are collapsed.
```

Edge lengths are compared to a fraction/factor of the target edge length, so "too long" and "too short" are more correct